src/overlays/Modal.scss

```diff
@@ -124,3 +126,20 @@
   justify-content: center;
   min-height: var(--bloom-s48);
 }
+
```

Comment:
I'm finding this implementation a bit confusing since setting --actions-display-mobile doesn't seem to affect the mobile view. I think the variable would have to be used on line 139 instead.

Curious on getting more thoughts here from @emilyjablonski and @eajensenwa. Although using variables, the footer is being called in different spots which kind of puts consumer in a limited customization spot unless we wanted four css variables associated with this. I'm wondering if the behavior of always showing action on mobile but only after scrolling for desktop is desired for all modals @eajensenwa. If so, I think adding a new prop may be clearer here.

Reply:
I would lightly prefer one `.modal__footer` class that has the mobile media query and different values for display, since we already have the two variables. Will lean on Em's guidance for #2!

Reply:
@ColinBuyck I don't think we should assume that we want all modals to show action only after scrolling (desktop) because most modals don't take up this much real estate. Is there a way I can preview what this looks like on desktop if we keep behavior as-is (action always showing)?

Reply:
@eajensenwa Detroit public dev should provide an example of that behavior on multiple photos. Let me know if you're looking for something else though!

Reply:
@ColinBuyck gotcha, thanks! Lets treat this as a unique pattern, meaning that buttons positioned at bottom of scroll on desktop shouldn't be inherited by the modal component.

Reply:
That's helpful @eajensenwa! In that case @KrissDrawing, these changes will have to include the addition of a prop dictating the modal action behavior. Let me know if you have any questions!